server/lib/orcasite_web/router.ex

```diff
@@ -94,7 +94,7 @@ defmodule OrcasiteWeb.Router do
 
     forward("/", Absinthe.Plug.GraphiQL,
       schema: Module.concat(["OrcasiteWeb.Schema"]),
-      interface: :playground,
+      interface: :simple,
```

Comment:
Do you know what the differences are between the interfaces? I took a quick look and seems to have just about the same features

Reply:
Also why not [`:advanced`](https://hexdocs.pm/absinthe_plug/Absinthe.Plug.GraphiQL.html#module-interface-selection)? Are we not ready? 🤣